diagnosis/getkuberneteslogs.sh

```diff
@@ -98,20 +98,22 @@ ensureStorageAccountContainer()
     fi
 }
 
-uploadLogs() 
+uploadLogs()
 {
     echo "$(date +%Y%m%d%H%M%S)][INFO] Uploading log files to container ${SA_CONTAINER}"
     az storage blob upload-batch -d ${SA_CONTAINER} -s ${SA_DIR} --destination-path ${SA_CONTAINER_DIR} --pattern *.zip --account-name ${SA_NAME}
     if [ $? -ne 0 ]; then
         echo "$(date +%Y%m%d%H%M%S)][ERR] Error uploading log files to container ${SA_CONTAINER}"
         exit 1
     fi
+    
+    az storage blob upload-batch -d ${SA_CONTAINER} -s ${SA_DIR} --destination-path ${SA_CONTAINER_DIR} --pattern *.json --account-name ${SA_NAME}
```

Comment:
You need to move `apimodel` to `SA_DIR`. Check how it is done for `akse-versions.txt`